### PR TITLE
Operaads: support multiformat request

### PR DESCRIPTION
**Pull request by jizeyopera**

### CODE REVIEW

File: adapters/operaads/operaads.go [hunk 1]
```diff
@@ -19,6 +21,11 @@ type adapter struct {
 	epTemplate *template.Template
 }
 
+var (
+	errBannerFormatMiss = errors.New("Size information missing for banner")
+	errDeviceOrOSMiss   = errors.New("Impression is missing device OS information")
+)
```
Comment (SyntaxNode) [inline adapters/operaads/operaads.go:27]: Nice use of package scoped errors.

**Review by hhhjort — APPROVED** [review]
Looks good to me.

**Comment by jizeyopera** [comment]
@bsardo Please help review. Thanks.

**Review by bsardo — APPROVED** [review]
LGTM

**Pull request merged**